### PR TITLE
List of sites broken by DPI protection

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/123450

Related to https://github.com/AdguardTeam/CoreLibs/issues/1645

### CODE REVIEW

File: EnglishFilter/sections/whitelist_stealth.txt [hunk 1]
```diff
@@ -42,6 +42,8 @@
 !
 !
 !
+! bcee.snet.lu - DPI feature of the Adguard Stealth break the website
+@@||bcee.snet.lu^$stealth
```
Comment (zloyden) [inline EnglishFilter/sections/whitelist_stealth.txt:46]: It should be reported here https://github.com/AdguardTeam/CoreLibs/issues/1645

Comment (Alex-302) [reply EnglishFilter/sections/whitelist_stealth.txt:46]: `$stealth` exclusion also should be added, and mentioned in CL issue(disable Base filter before check)

Comment (zloyden) [reply EnglishFilter/sections/whitelist_stealth.txt:46]: `DPI feature of the Adguard Stealth break the website`
I think it's better to be
```
[Stealth Mode - Protect from DPI]
```

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/whitelist_stealth.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/whitelist_stealth.txt:46]: Also add other domains from the task https://github.com/AdguardTeam/CoreLibs/issues/1645

File: EnglishFilter/sections/whitelist_stealth.txt [hunk 2]
```diff
@@ -42,6 +42,15 @@
 !
 !
 !
+! TODO: remove when this issue will be fixed - https://github.com/AdguardTeam/CoreLibs/issues/1645
+! [Stealth Mode - Protect from DPI]
```
Comment (Alex-302) [inline EnglishFilter/sections/whitelist_stealth.txt:46]:
```suggestion
! https://github.com/AdguardTeam/CoreLibs/issues/1645 [Stealth Mode - Protect from DPI]
```

File: EnglishFilter/sections/whitelist_stealth.txt [hunk 3]
```diff
@@ -42,6 +42,15 @@
 !
 !
 !
+! TODO: remove when this issue will be fixed - https://github.com/AdguardTeam/CoreLibs/issues/1645
```
Comment (Alex-302) [inline EnglishFilter/sections/whitelist_stealth.txt:45]:
```suggestion
! TODO: remove when this issue will be fixed
```

**Review by Alex-302 — CHANGES_REQUESTED** [review]
```
! https://github.com/AdguardTeam/CoreLibs/issues/1645 [Stealth Mode - Protect from DPI]
! TODO: remove when this issue will be fixed
```

File: EnglishFilter/sections/whitelist_stealth.txt [hunk 4]
```diff
@@ -42,6 +42,17 @@
...
+! https://github.com/AdguardTeam/CoreLibs/issues/1645 [Stealth Mode - Protect from DPI]
+! TODO: remove when this issue will be fixed
+!#if (adguard_app_windows || adguard_app_mac || adguard_app_android)
+@@||bcee.snet.lu^$stealth
+@@||pravo.by^$stealth
+@@||kun.uz^$stealth
+@@||mfah.org^$stealth
+@@||aztelekom.az^$stealth
+@@||gosuslugi.ru^$stealth
+@@||filecity.co.kr^$stealth
```
Comment (zloyden) [inline EnglishFilter/sections/whitelist_stealth.txt:54]: In fact, it's not a complete list from the [task](https://github.com/AdguardTeam/CoreLibs/issues/1645) and comments. What about the others?

**Review by Alex-302 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**